compiler/optimizer/GlobalValuePropagation.hpp

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2000, 2016
```

Comment:
Looking at the changes now... But as for the year, it should be changed to 2017 for all the files changed, not just the new files.